tools/tracer/README.md

```diff
@@ -0,0 +1,46 @@
+Copyright(c) 2014-2020 Intel Corporation. All Rights Reserved.
```

Comment:
document is badly rendered: https://github.com/egorovdanil/MediaSDK/blob/master/tools/tracer/README.md. Please, fix it.

Reply:
done